docs/ko/docs/tutorial/middleware.md

```diff
@@ -0,0 +1,61 @@
+# 미들웨어
+
+미들웨어를 **FastAPI** 애플리케이션에 추가할 수 있습니다.
+
+"미들웨어"는 특정 *경로 동작*에 의해 처리되기 전, 모든 **요청**에 대해서 동작하는 함수입니다. 또한 모든 **응답**이 반환되기 전에도 동일하게 동작합니다.
+
+* 미들웨어는 애플리케이션으로 오는 **요청**를 가져옵니다.
+* **요청** 또는 다른 필요한 코드를 실행 시키는 동작을 할 수 있습니다.
+* **요청**을 애플리케이션의 *경로 작업*으로 전달하여 처리합니다.
+* 애플리케이션의 *경로 작업*에서 생성한 **응답**를 받습니다.
+* **응답** 또는 다른 필요한 코드를 실행시키는 동작을 할 수 있습니다.
+* **응답**를 반환합니다.
+
+!!! note "기술 세부사항"
+만약 `yield`를 사용한 의존성을 가지고 있다면, 미들웨어가 실행되고 난 후에 exit이 실행됩니다.
+
+    만약 (나중에 문서에서 다룰) 백그라운드 작업이 있다면, 모든 미들웨어가 실행되고 *난 후에* 실행됩니다.
+
+## 미들웨어 만들기
+
+미들웨어를 작성하기 위해서 함수 상단에 `@app.middleware("http")` 데코레이터를 사용할 수 있습니다.
+
+미들웨어 함수는 다음 항목들을 받습니다:
+
+* `요청`.
+* `요청`를 매개변수로 받는 `call_next` 함수.
+    * 이 함수는 `요청`를 해당하는 *경로 작업*으로 전달합니다.
+    * 그런 다음, *경로 작업*에 의해 생성된 `응답` 를 반환합니다.
+* `응답`를 반환하기 전에 추가로 `응답`를 수정할 수 있습니다.
```

Comment:
> Because this is not a special word but emphasizes syntax for markdown and just HTTP request, you need to translate `**request**`😁.


hmm... you said that the keyword "request" and "response" should be translated.
but in this review, your suggestion said that keyword "request" and "response" use in eng.
음.. 처음에 해주신 request, response에서는 request와 response가 해석이 되어야 한다고 리뷰를 하셨습니다. 그러나, 이번 리뷰에서는 request, response를 해석하지 않으신 것을 제안 하셨어요!


So, I'd like to suggest that if a term usually use in english at our business hours, use English at translate contribution.
그래서 일반적으로 현업에서 영어 원본 그대로 많이 쓰는 영어 그대로 두는 것을 컨벤션으로 지정하는 것에 대해 제안드리고 싶어요!